includes/class-sensei-grading-user-quiz.php

```diff
@@ -12,43 +18,60 @@
  * @since 1.3.0
  */
 class Sensei_Grading_User_Quiz {
-	public $user_id;
-	public $lesson_id;
-	public $quiz_id;
+	/**
+	 * The user id which this quiz is for.
+	 *
+	 * @var int
+	 */
+	private $user_id;
+
+	/**
+	 * The lesson id which the quiz belongs to.
+	 *
+	 * @var int
+	 */
+	private $lesson_id;
 
 	/**
-	 * Constructor
+	 * The quiz id.
+	 *
+	 * @var int
+	 */
+	private $quiz_id;
+
+	/**
+	 * Sensei_Grading_User_Quiz constructor.
 	 *
 	 * @since  1.3.0
+	 *
+	 * @param int $user_id  The user id.
+	 * @param int $quiz_id  The quiz id.
 	 */
 	public function __construct( $user_id = 0, $quiz_id = 0 ) {
 		$this->user_id   = intval( $user_id );
 		$this->quiz_id   = intval( $quiz_id );
 		$this->lesson_id = get_post_meta( $this->quiz_id, '_quiz_lesson', true );
-	} // End __construct()
+	}
 
 	/**
-	 * build_data_array builds the data for use on the page
-	 * Overloads the parent method
+	 * Builds the data for use on the page.
 	 *
 	 * @since  1.3.0
 	 * @return array
 	 */
-	public function build_data_array() {
-		$data_array = Sensei_Utils::sensei_get_quiz_questions( $this->quiz_id );
-		return $data_array;
-	} // End build_data_array()
+	private function build_data_array() {
```

Comment:
I don't think we should change a `public` function to a `private` one as it could break custom code.

Reply:
c73913a55f49492a63b34b1c7c25b4c5bf57d3ce